abcix/types/application.go

```diff
@@ -4,6 +4,9 @@ import (
 	context "golang.org/x/net/context"
 )
 
+type MempoolIter interface {
```

Comment:
leaving it empty is for not breaking existing code?

Reply:
I try to add `mempl.Mempool` here, but an error said import cycle, still working on that.

Reply:
no we said it should only be a subset of `Mempool` interface, which means you only need to add one method inside this iterator, which is the one added in #10